demo/midgard-node/src/services/always-succeeds.ts

```diff
@@ -62,29 +54,74 @@ export const makeAuthenticatedValidator = (
 
 const makeAlwaysSucceedsService: Effect.Effect<
   {
+    hubOracleAuthValidator: AuthenticatedValidator;
```

Comment:
After removing the spending script, you most likely won't want to use `AuthenticatedValidator` for hub oracle.